src/sentry/integrations/gitlab/integration.py

```diff
@@ -181,11 +181,10 @@ def get_commit_context(
         if not commitInfo:
             return None
         else:
-            committed_date = "{}Z".format(
-                datetime.strptime(commitInfo.get("committed_date"), date_format_expected)
-                .astimezone(timezone.utc)
-                .strftime("%Y-%m-%dT%H:%M:%S.%f")[:-3]
-            )
+            committed_date = datetime.strptime(
+                commitInfo.get("committed_date"), date_format_expected
+            ).astimezone(timezone.utc)
```

Comment:
this is sad that we have to define the format for 8601. Can we put a comment to replace this when we get to python 3.11 and can use `dateutil.parser.isoparse`

Reply:
there is `isodate.parse_date` but we don't seem to use it anywhere

Reply:
https://github.com/gweis/isodate/#documentation